src/Commands/AbstractDrupalCommands.php

```diff
@@ -378,6 +379,19 @@ public function settingsSetup(array $options = [
             $this->getConfig()
         )->setConfigKey('drupal.settings');
 
+        // If ran in dev mode copy local settings file to 'settings.local.php'.
```

Comment:
Well it's not the default local settings file that gets copied, its whatever configured in runner.yml:
```
    # Local development settings override.
    # Add development settings that you want to share with your team and commit this file.
    # If found this file will be copied in `/sites/{sites-subdir}/settings.local.php`,
    # which should be, instead, git-ignored. This will give the possibility to locally
    # tweak development settings.
    settings_local_file: "${drupal.root}/sites/example.settings.local.php"
```

I'll rename in local settings overrides.

Reply:
I'll leave it like that at the end.